man/make_man.sh

```diff
@@ -46,6 +46,22 @@ subsystem and devices
 .sp
 cc file.c
 .B -liio
+.SH OPTIONS
+The define
+.B
+IIO_CHECK_REG
+will warn if return values are not checked. Most
+.B
+libiio
+functions, if/when a failure occurs will return a negative error number.
+this warning will ensure these error numbers are looked it. There is
```

Comment:
shouldn't it be: "are looked at" ?

Reply:
fat fingers and lazy eyes, fixed now. Thanks for checking.